src/args/args.go

```diff
@@ -23,4 +23,5 @@ type ArgumentList struct {
 	VhostsRegexes        string `default:"" help:"JSON array of vhost name regexes from which to collect metrics."`
 	ShowVersion          bool   `default:"false" help:"Print build information and exit"`
 	Timeout              int    `default:"30" help:"Timeout in seconds to timeout the connection to RabbitMQ endpoint."`
+	DisableEntities      bool   `default:"false" help:"configure whether inventory entries are created for entities."`
```

Comment:
what is the difference with the existing Inventory arg supported by the sDK? Is just the name being more user-friendly or also was the code not supporting it? 
We could even think of linking somehow the two

Reply:
That disables the inventory collection (real inventory). This flags disables the adding of inventory entries to generate entities when collecting metrics (even if the inventory flag is disable)

Reply:
It could work to link to use the inventory actually , so when METRICS=true no inventory is sent, I will change it and I think we can remove the extra flag

Reply:
as discussed offline, using the INVENTORY flag to remove this will break existing configurations which doesn't make sense since the feature we want to add is a workaround for the entities generation on larger environments